### DISCUSSION TITLE
Don't see a results from pcap file in SO

### DISCUSSION THREAD

**Discussion by habibbarika** [Security-Onion-Solutions/securityonion#9353] [General] [answered]
I am very interested to security onion  to put it in my monitoring lab.
My question is :
I know that to start it is strongly advised to import pcap files and analyze them. Unfortunately when I import from Malware analysis I don't see alerts and nothing in Hunt, this doesn't happen all the time. This is my pcap file :

[2020-07-20-IcedID-infection-traffic.pcap.zip](https://www.malware-traffic-analysis.net/2020/07/20/2020-07-20-IcedID-infection-traffic.pcap.zip)   5.9 MB (5,916,514 bytes)

Another question 👍 

What is the difference between AF_Packet and Raw_packet and what's the type of packet supported bye Wireshark. Because I see in architecture the tree components support AF_Packet : Suricata, Zeek and Steno. 
[File.docx](https://github.com/Security-Onion-Solutions/securityonion/files/10201222/File.docx)

Thank you !!!

**Comment by dougburks** [comment]
> I know that to start it is strongly advised to import pcap files and analyze them. Unfortunately when I import from Malware analysis I don't see alerts and nothing in Hunt, this doesn't happen all the time. This is my pcap file :

Works fine for me:
![image](https://user-images.githubusercontent.com/1659467/207042929-c74be7e4-b146-4d2e-8c5e-3cf37467e8aa.png)

Make sure you are using the hyperlink provided by so-import-pcap or manually setting your time range correctly.

> What is the difference between AF_Packet and Raw_packet and what's the type of packet supported bye Wireshark. Because I see in architecture the tree components support AF_Packet : Suricata, Zeek and Steno.

From https://docs.securityonion.net/en/2.3/af-packet.html:
_AF-PACKET is built into the Linux kernel and includes fanout capabilities enabling it to act as a flow-based load balancer. This means, for example, if you configure Suricata for 4 AF-PACKET threads then each thread would receive about 25% of the total traffic that AF-PACKET is seeing._

For more information about AF-PACKET, please see:
https://www.kernel.org/doc/Documentation/networking/packet_mmap.txt

**Reply by habibbarika** [reply]
Thank you very much Doug. Always dynamic, continuous!

**Reply by habibbarika** [reply]
I downloaded the following pcap: 
![image](https://user-images.githubusercontent.com/120285406/208471937-22e54352-96de-47f4-80a6-9ecd95d2ac93.png)


I changed the schedule and used your solution with the /tmp folder to help Suricata access, but nothing appears!! It's weird !

![SO-Pcap](https://user-images.githubusercontent.com/120285406/208473258-1f25da41-dea9-4046-a181-b4da16422521.PNG)

thank you again !

**Reply by dougburks** [reply]
Looking at your screenshot, it says `this PCAP has already been imported; skipping`. If you go to Hunt and change the time frame to start at 2020/05/27 and end at 2020/05/29, does it return any events?

If you really want to import the pcap again, you should be able to delete the subdirectories under `/nsm/import/` and then run `so-import-pcap` again.

**Reply by habibbarika** [reply]
Hi Doug !

I’m starting over :

![image](https://user-images.githubusercontent.com/120285406/209032998-2dc5ec3b-7c7e-48d4-8f11-e8734945cafd.png)

Import again :

![image](https://user-images.githubusercontent.com/120285406/209033029-3d14e84a-574d-43cf-a64e-7ca367646cd0.png)

Importation completed !
![image](https://user-images.githubusercontent.com/120285406/209033061-43d2f7f0-f479-489f-9627-11bb83114e66.png)

All service start correctly :

![image](https://user-images.githubusercontent.com/120285406/209033108-bb864434-9378-4923-bc5a-743a566a9f16.png)
Message error when I lunch SOC Web interface. Execute soup command two times:
![image](https://user-images.githubusercontent.com/120285406/209033149-8ff2b5d1-e350-4c5d-87ee-335bcb621bd8.png)
Many updates! More than 30 minutes! More than a complete installation
![image](https://user-images.githubusercontent.com/120285406/209033180-c88cd1f8-b986-414b-b4cf-4391fea6b530.png)
![image](https://user-images.githubusercontent.com/120285406/209033207-7c41819a-1733-425e-bcc0-b8e1b079ac1f.png)
![image](https://user-images.githubusercontent.com/120285406/209033220-9f799d09-2715-47aa-affa-e0bb9e36cf0f.png)
Access to SOC: after update the same error
![image](https://user-images.githubusercontent.com/120285406/209033262-ed1a8738-129c-40ff-ab5d-27bcbb2773c6.png)
I reinstall!!!    Looks like I'm having bad luck with security onion, yet I love it!
![image](https://user-images.githubusercontent.com/120285406/209033305-b93ab84e-10b5-4ea1-a817-c21ba7c18732.png)

I apologize for the length of my post! :(

Thanks very much !

**Reply by dougburks** [reply]
I'd recommend downloading the latest version 2.3.190 and performing a fresh installation:
https://github.com/Security-Onion-Solutions/securityonion/blob/master/VERIFY_ISO.md

If you're just going to be importing pcap files, then you can just do an import installation which will be much quicker and easier:
https://docs.securityonion.net/en/2.3/first-time-users.html

**Comment by habibbarika** [comment]
Hi Doug,

I reinstalled SO with import mode:
![image](https://user-images.githubusercontent.com/120285406/209458520-e0a27292-dcb1-443b-80a3-69920506e38e.png)
![image](https://user-images.githubusercontent.com/120285406/209458528-168fda7b-214a-4bde-9473-8519591f5dc3.png)

I imported the same file as you:
![image](https://user-images.githubusercontent.com/120285406/209458534-decb4be4-ad86-4233-876d-7d566e5d1f62.png)
I modified the date :
![image](https://user-images.githubusercontent.com/120285406/209458549-1be961dc-915c-45da-b9cb-99bb6823e614.png)

In Hunt :
![image](https://user-images.githubusercontent.com/120285406/209458557-a47d2364-0010-451c-8d7f-65255b3f4939.png)

Thank you !!!

**Reply by dougburks** [reply]
If you have no NIDS alerts, then check the NIDS troubleshooting steps:
https://docs.securityonion.net/en/2.3/suricata.html#troubleshooting-alerts

**Comment by habibbarika** [comment]
Hi Doug,

I couldn't fix my PCAP problem, it's disheartening to start with this problem. I plan to teach it to my students, but it's off to a bad start. It's rare or I get stuck like that, I always find a solution to my problems.

As shown in my drawing, I am unable to ping from Kali to SO. Also, I can't capture traffic from kali attack on Windows 10 (Simple scan nmap -O -sV). Why ?

![image](https://user-images.githubusercontent.com/120285406/211172759-322724f5-397d-4d10-8b72-a5c7266da7fb.png)

Anyway, thank you for your input!

**Reply by dougburks** [reply]
> As shown in my drawing, I am unable to ping from Kali to SO.

Seems like a networking issue. Make sure that your Security Onion installation has a management interface in the same subnet as your Kali box and that there are no firewalls or other network devices that may be interfering with the traffic.

> Also, I can't capture traffic from kali attack on Windows 10 (Simple scan nmap -O -sV).

If you've configured Security Onion for Import mode, then it doesn't support live traffic capture. Otherwise, this could be an issue with your tap, span, or virtualization platform.

**Comment by habibbarika** [comment]
Hi Doug,

If I understand correctly (so) must have a card for monitoring and another for management. Me, what I want to know, if it is possible to ping the monitoring card. I want to make sure that there is communication in my local network. On my monitoring network card, I activated promiscuous mode (.vmx file). Of course my bridge network, where there is my management card is to access the SOC and see the traffic to analyze. The ping to my management card is working fine, from my host machine.
For my installation, it is the EVAL version.

Thank you for your answers !!!

**Reply by dougburks** [reply]
From https://docs.securityonion.net/en/2.3/hardware.html#nic:
> If you plan to sniff network traffic from a tap or span port, then you will need one or more interfaces dedicated to sniffing (no IP address).

**Comment by habibbarika** [comment]
Hi Doug,
A question about sysmon!
Does the file you created in Git cover all events supported by sysmon or not. How many events supported by sysmon without configuration file ?
![image](https://user-images.githubusercontent.com/120285406/212599346-95208436-8304-4be2-8aa1-32909fdd85da.png)

![image](https://user-images.githubusercontent.com/120285406/212598789-dd3e3ce3-4583-447f-b546-2b1c12f7dc48.png)

Is the list of events limited to the following table:

![image](https://user-images.githubusercontent.com/120285406/212599018-f586b21a-a13a-47c8-8057-67a1afb41bec.png)

Thank you Doug !

**Reply by dougburks** [reply]
> Does the file you created in Git cover all events supported by sysmon or not.

I didn't create the sysmon config. Per your screenshot, it was created by SwiftOnSecurity:
https://github.com/SwiftOnSecurity/sysmon-config

> Is the list of events limited to the following table:

You can read more about Sysmon events at:
https://learn.microsoft.com/en-us/sysinternals/downloads/sysmon#events

This discussion has strayed from its original title of `Don't see a results from pcap file in SO`. If you have further questions or problems, please start a new discussion with an appropriate title.

**Comment by habibbarika** [comment, marked as answer]
Hi Doug,

Finally, I found my problem concerning the detection of alerts with Securita. I was just missing an update with the soup command. I'm happy, I can move forward to make the product usable in the French-speaking community.

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/120285406/02d749e5-ae7c-42fd-a194-6c99d8b5c080)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/120285406/6a6b1ca6-68d5-440b-9d3f-b87a9e3b6197)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/120285406/559d357c-6536-4e6f-bb66-9916014a8ed6)
 
Thank you so much !

**Reply by bdavedu** [reply]
Hi,

I have the same problem. When I import a PCAP file via SOC or via CLI, I only can see Dashboard for this PCAP but when I go to alerts and filter by dates there aren't any alert. How did you solve it ?

**Reply by dougburks** [reply]
@bdavedu From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and in that new discussion you can provide a hyperlink to the related discussion.
